buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.sh

```diff
@@ -0,0 +1,612 @@
+    fi
+
+    FOUND_ORACLE_COPYRIGHT=$(head -n $MAX_LINES "$1" | $GREP -n "Copyright (c).*Oracle" | head -n 1 | cut -d: -f1)
+    [ -z "$FOUND_ORACLE_COPYRIGHT" ] && FOUND_ORACLE_COPYRIGHT=0
+    if [ "$FOUND_ORACLE_COPYRIGHT" -gt 0 ]; then
+      trace "We have found an Oracle Copyright on line $FOUND_ORACLE_COPYRIGHT in file: $1"
+    fi
+    FOUND_COPYRIGHT=0
+    if [ "$FOUND_IBM_COPYRIGHT" -eq 0 ]; then
+      if [ "$FOUND_APACHE_COPYRIGHT" -eq 0 ]; then
+        if [ "$FOUND_BSD_OR_MIT_COPYRIGHT" -eq 0 ]; then
+          if [ "$FOUND_ORACLE_COPYRIGHT" -eq 0 ]; then
+            FOUND_COPYRIGHT=$(head -n $MAX_LINES "$1" | $GREP -n "Copyright" | head -n 1 | cut -d: -f1)
+          fi
+        fi
+      fi
+    fi
+    [ -z "$FOUND_COPYRIGHT" ] && FOUND_COPYRIGHT=0
+    if [ "$FOUND_COPYRIGHT" -gt 0 ]; then
+      trace "We have found a different Copyright on line $FOUND_COPYRIGHT in file: $1"
+    fi
+
+    FOUND_IBM_CPE=0
+    FOUND_ORACLE_CPE=0
+    FOUND_CPE=$(head -n $MAX_LINES "$1" | $GREP -n " \"Classpath\"" | head -n 1 | cut -d: -f1)
+    [ -z "$FOUND_CPE" ] && FOUND_CPE=0
+    if [ "$FOUND_CPE" -eq 0 ]; then
+      FOUND_CPE=$(head -n $MAX_LINES "$1" | $GREP -n -i " Classpath Exception" | head -n 1 | cut -d: -f1)
+      [ -z "$FOUND_CPE" ] && FOUND_CPE=0
+    fi
+    if [ "$FOUND_CPE" -gt 0 ]; then
+      trace "We have found a Classpath Exception on line $FOUND_CPE in file: $1"
+      if [ "$FOUND_IBM_DESIGNATES" -gt 0 ]; then
+        FOUND_IBM_CPE=$FOUND_CPE
+        trace "We have found the IBM CPE on line $FOUND_ORACLE_CPE in file: $1"
+      else
+        if [ "$FOUND_ORACLE_DESIGNATES" -gt 0 ]; then
+          FOUND_ORACLE_CPE=$FOUND_CPE
+          trace "We have found the Oracle CPE on line $FOUND_ORACLE_CPE in file: $1"
+        fi
+      fi
+    fi
+
+    FOUND_GPLV2=$(head -n $MAX_LINES "$1" | $GREP -n "GNU General Public License" | head -n 1 | cut -d: -f1)
+    [ -z "$FOUND_GPLV2" ] && FOUND_GPLV2=0
+    if [ "$FOUND_GPLV2" -gt 0 ]; then
+      trace "We have found GPLv2 on line $FOUND_GPLV2 in file: $1"
+    fi
+
+    FOUND_NON_IBM_COPYRIGHT=0
+    [ "$FOUND_APACHE_COPYRIGHT" -gt 0 ]     && FOUND_NON_IBM_COPYRIGHT=1
+    [ "$FOUND_BSD_OR_MIT_COPYRIGHT" -gt 0 ] && FOUND_NON_IBM_COPYRIGHT=1
+    [ "$FOUND_ORACLE_COPYRIGHT" -gt 0 ]     && FOUND_NON_IBM_COPYRIGHT=1
+    [ "$FOUND_COPYRIGHT" -gt 0 ]            && FOUND_NON_IBM_COPYRIGHT=1
+    if [ "$FOUND_NON_IBM_COPYRIGHT" -eq 1 ]; then
+      trace "We have found a non IBM copyright"
+    fi
+
+    # Check to see whether the file is in the built JDK
+    IN_JDK=1
+    case "$1" in
+#      */solaris/*)
+#        echo  "$1 deemed not to be in the built JDK because it matches case parameter expansion */solaris/*"
+#        IN_JDK=0;;
+#      *make/*)
+#        echo  "$1 deemed not to be in the built JDK because it matches case parameter expansion *make/*"
+#        IN_JDK=0;;
```

Comment:
Just confirming if these 2 cases are commented out intentionally

Reply:
Yes.  Turned out that only some of the files in that pattern should be excluded, so more specific patterns were used for those.  Removed the commented out lines.